src/psyclone/tests/dependency_test.py

```diff
@@ -159,7 +159,7 @@ def test_if_statement(parser):
     var_accesses = VariablesAccessInfo()
     if_stmt.reference_accesses(var_accesses)
     assert str(var_accesses) == "a: READ, b: READ, i: READ, p: WRITE, "\
-                                "q: READWRITE, r: READ"
+                                "q: READ+WRITE, r: READ"
```

Comment:
What's the difference between `READWRITE` and `READ+WRITE` and please could you document this in the Developers' Guide?

Reply:
I am adding this to the documentation of the __str__ method (which is included in the developers guide), since there is no separate type 'READ+WRITE'. I am not even sure if this is really required - it allows to distinguish between:
```
a=a+1     ! one read and one write access with same location
b=a; a=7;    ! one read and one write access with different location
call sub(a)    ! with metadata for a indicating READWRITE
```
And I thought it would be useful to see the difference between the first two and the last access in __str__().

Reply:
That's great - thanks.